CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contribution Guidelines
+
+Thanks for your interest in contributing to dwave-examples! Before getting
+started, we encourage you to take a look at this document for some guidelines.
+
+## What We're Looking For
+
+While we have a preference for application oriented examples, we welcome all
+examples that demonstrate use of the D-Wave system.
```

Comment:
Is this accurate? "We welcome all examples that demonstrate use of ____"?

Reply:
"D-Wave solvers"? Would we accept "demonstrate use of Ocean tools" in general?

Reply:
I'm okay with "demonstrate use of Ocean tools"